addons/project/static/tests/burndown_chart_tests.js

```diff
@@ -46,6 +46,15 @@ QUnit.module("Project", {}, () => {
                             { id: 2, name: "In Progress" },
                             { id: 3, name: "Done" },
                         ],
+                    },
+                    "project.task.type": {
+                        fields: {
+                            name: { string: "Name", type: "char" },
+                            sequence: { type: "integer"}
+
+                        },
+                        records: []
+
```

Comment:
```suggestion
                    },
                    "project.task.type": {
                        fields: {
                            name: { string: "Name", type: "char" },
                            sequence: { type: "integer"}
                        },
                        records: []
```

Reply:
But sounds weird. Code what you undersand :laughing:

Reply:
The 'serverData' object seems to become the backend data, when fetching data from the frontend in test, the data is fetched from this object.
So when i do the 
```
    async _fetchStageSeq(searchParams) {
        const data = await this.orm.webSearchRead(
            "project.task.type",
            [["project_ids", "in", searchParams.context.active_id]],
            ["name", "sequence"]
        );
```
The serverData needs to have some 'project.task.type' key.
The problem is that i would like it to be the burndown_chart.stage_id but i didn't find how to do this, i need help on this one ^^'
When fetching the points in the _graph_model.js_
```
async _loadDataPoints(metaData) {
...
this.orm
  .webReadGroup(
      resModel,
      domain.arrayRepr,
      measures,
      groupBy.map((gb) => gb.spec),
      {
          lazy: false, // what is this thing???
          context: { fill_temporal: true, ...this.searchParams.context },
      }
  )
...
}
```

the 'resModel' is changed from 'project.task.burndown.chart.report' ( in production ) to 'burndown_chart' ( in testing ). 
This is done somewhere in the 
```
await makeView(makeViewParams);
```
in _burndown_chart_tests.js_ but i couldn't find how, tell me if you have some insights about this